src/assets/styles/base/_base.scss

```diff
@@ -31,14 +31,14 @@ a {
 main,
 footer {
   a {
-    border-radius: 5px;
+    border-radius: rem(5);
   }
 
   a:hover,
   a:focus,
   a:active {
     outline-style: solid;
-    outline-width: 2px;
+    outline-width: rem(3);
```

Comment:
This should be 2, right? Looks like a lot of these are 3 instead of 2.

Reply:
I noticed that I used 3 for the resources page work before, so I changed it to 3 to make it consistent, but I think we can do 2 if that's what we want.

Reply:
Looking at the other links now, I think you're right and it should be 3!